conf/location-ada-cs-errors.conf

```
@@ -6,13 +6,13 @@ error_page 403 /ada_cs_forbidden.html;
 
 location = /ada_cs_error.html {
     include conf/headers-default.conf;
-    add_header Content-Security-Policy "default-src 'none'; script-src 'unsafe-inline' https://www.google-analytics.com; img-src 'self' data: https://www.google-analytics.com; object-src 'none'; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com; font-src 'self' https://fonts.gstatic.com;" always;
+    add_header Content-Security-Policy "default-src 'none'; script-src 'none; img-src 'self'; object-src 'none'; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com; font-src 'self' https://fonts.gstatic.com;" always;
```

```suggestion
    add_header Content-Security-Policy "default-src 'none'; img-src 'self'; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com; font-src 'self' https://fonts.gstatic.com;" always;
```
`script-src: 'none` was missing its closing quote, but is also unnecessary since it will inherit the `default-src` if unset, and that is also `'none'`.
By the same logic, `object-src` could go too; save us some bytes and make it easier to read.

---

Damn! Thanks!